content/applications/marketing/survey.rst

```diff
@@ -1,8 +1,304 @@
+
+.. image:: survey/time-limit-survey-clock.png
+   :align: center
+   :alt: Survey time limit clock display front-end
+
+.. note::
+   Surveys that are not submitted in time **do not** have their answers saved.
+
+Randomized surveys
+******************
+
+When a survey is randomized, the questions will be shuffled in a random order every time a
+participant opens the survey. Random surveys are a great way to eliminate the possibility of
+participants copying off one another.
+
+Once you've enabled that feature, go to the *Questions* tab of the survey, where a
+*Random questions count* column is now present.
+
+Here you can decide how many of the questions should be taken into account during the shuffling,
+by simply selecting the number and changing it. That number corresponds to how many questions in
+that section will be randomized.
+
+.. image:: survey/random-questions-count.png
+   :align: center
+   :alt: Survey random questions count
+
+Scoring
+~~~~~~~
+
+Surveys are a great way to measure a person's performance, knowledge of a subject, or overall
+satisfaction.
+
+Odoo allows you to attach points to certain answers of survey questions. These points are then
+summed up to give your participant a final score.
+
+Users have the option to choose between: *Scoring with answers at the end* **or** *Scoring without
+answers at the end*.
+
+If you select either *Scoring with answers at the end* or *Scoring without answers at the end*, two
+more fields appear. In which, you decide what the *Success %* is, and you can choose to make this a
+certification.
+
+.. image:: survey/survey-options-scoring.png
+   :align: center
+   :alt: Survey option template tab
+
+You can set the *Success %* here, as well. The *Success %* is the score the user will need to
+achieve in order to have successfully taken the survey.
+
+.. image:: survey/survey-success-percentage.png
+   :align: center
+   :alt: Survey success percentage setting
+
+If *Is a Certification* is selected, you can choose its template. The certification is automatically
+emailed to users who have successfully finished the survey.
+
+.. image:: survey/survey-certification-template-setting.png
+   :align: center
+   :alt: Survey certification template setting
+
+And, if you check the box next to *Give Badge,* you are able to select which Certification Badge
+you want to give to your participants.
+
+.. image:: survey/survey-scoring-section.png
+   :align: center
+   :alt: Scoring section of survey settings
+
+.. note::
+   The *Give Badge* feature is **only** available if you enable *Login required* in the
+   *Candidates* section of the *Options* tab of the survey detail form.
+
+Following those configurations, you can choose a survey question that you want to assign a point
+value to, and mark which answer is "correct" on the question pop-up. You can also apply negative
+point results for "incorrect" answers, as well.
```

Comment:
I actually think it works well in its current position, but if you think it would be best to relocate it, I'll be more than happy to do so. Thanks again for all this amazing feedback. It's extremely helpful.